lib/ssl/src/tls_record.erl

```diff
@@ -112,12 +112,18 @@ encode_handshake(Frag, {3, 4}, ConnectionStates) ->
 encode_handshake(Frag, Version, 
 		 #{current_write :=
 		       #{beast_mitigation := BeastMitigation,
+                         max_fragment_length := MaxFragmentLength,
 			  security_parameters :=
 			     #security_parameters{bulk_cipher_algorithm = BCA}}} = 
 		     ConnectionStates) ->
+    MaxLength = if is_integer(MaxFragmentLength) ->
```

Comment:
This code block appears twice, it could be factored into a separate function.

Reply:
I looked it up, this code block appears 5 times. It would be better if they were factored into a function.

```shell
tls_record_1_3.erl
48:    MaxLength = if is_integer(MaxFragmentLength) ->
49-                        MaxFragmentLength;
50-                   true ->
51-                        %% TODO: Consider padding here
52-                        ?MAX_PLAIN_TEXT_LENGTH
--
80:    MaxLength = if is_integer(MaxFragmentLength) ->
81-                        MaxFragmentLength;
82-                   true ->
83-                        ?MAX_PLAIN_TEXT_LENGTH
84-                end,

tls_record.erl
119:    MaxLength = if is_integer(MaxFragmentLength) ->
120-                        MaxFragmentLength;
121-                   true ->
122-                        ?MAX_PLAIN_TEXT_LENGTH
123-                end,
--
168:    MaxLength = if is_integer(MaxFragmentLength) ->
169-                        MaxFragmentLength;
170-                   true ->
171-                        ?MAX_PLAIN_TEXT_LENGTH
172-                end,

dtls_record.erl
220:    MaxLength = if is_integer(MaxFragmentLength) ->
221-                        MaxFragmentLength;
222-                   true ->
223-                        ?MAX_PLAIN_TEXT_LENGTH
224-                end,
```